config/default/kustomization.yaml

```diff
@@ -9,8 +9,12 @@ namespace: kyma-system
 namePrefix: istio-
 
 # Labels to add to all resources and selectors.
-commonLabels:
-  app.kubernetes.io/component: istio-operator.kyma-project.io
+
+labels:
+  - includeSelectors: false
+    pairs:
+      kyma-project.io/module: istio
+      app.kubernetes.io/version: dev
```

Comment:
For now this version can be bumped manually instead of the VERSION file. I'm not a fan of having dirty HEAD by modifying a branch during workflow runtime.

Reply:
If you do anything to the codebase in the release workflow, you break the rule of releasing what's in the revision. I know that you don't like the manual steps, but then you will have to agree that we will eventually break this rule and have dirty HEAD during release.

I'll just port what's been implemented in api-gateway ¯\_(ツ)_/¯

Reply:
I like ```$(KUSTOMIZE) edit set label [app.kubernetes.io/version:bar](http://app.kubernetes.io/version:bar)``` solution much better that the `sed` templating, will adapt in api-gateway as well.